### PR TITLE
Refactor view logic into helpers and partials

### PR DESCRIPTION
**Pull request by uryelah**
This PR removes the view logic at the scaffold views and refactors them into partials and helper methods.

As tested by me and Shamim all is still working properly.

Originally from issue #156.

CC @bolah2009

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -17,6 +17,8 @@
 default: &default
   adapter: postgresql
   encoding: unicode
+  username: postgres
+  password: <%= ENV['ROR_SOCIAL_SCAFFOLD_DATABASE_PASSWORD'] %>
```
Comment (bolah2009) [inline config/database.yml:21]: Are you sure this won't break for different users? I don't think we need to add username and password.

File: Gemfile.lock [hunk 2]
```diff
@@ -226,10 +235,11 @@ DEPENDENCIES
   turbolinks (~> 5)
   tzinfo-data
   uglifier (>= 1.3.0)
+  wdm (>= 0.1.0)
   web-console (>= 3.3.0)
 
 RUBY VERSION
-   ruby 2.7.0p0
+   ruby 2.6.6p146
```
Comment (bolah2009) [inline Gemfile.lock:242]: Do we need to downgrade the ruby version?

Comment (uryelah) [reply Gemfile.lock:242]: My bad I forgot to change the version back. 
I'll edit it back to 2.7

File: config/database.yml (hunk 1 above)
Comment (uryelah) [reply config/database.yml:21]: No I'm not, I'll remove it.

**Comment by uryelah** [comment]
@bolah2009 review fixes done 🌱

**Comment by bolah2009** [comment]
Thanks, @uryelah

Everything working fine as expected. LGTM

**Pull request merged**